src/cdk/layout/breakpoints-observer.ts

```diff
@@ -58,10 +58,15 @@ export class BreakpointObserver implements OnDestroy {
     let queries = coerceArray(value);
     let observables = queries.map(query => this._registerQuery(query).observable);
 
-    return combineLatest(observables, (a: BreakpointState, b: BreakpointState) => {
-      return {
-        matches: !!((a && a.matches) || (b && b.matches)),
-      };
+    return combineLatest<BreakpointState>(observables, (...states: BreakpointState[]) => {
```

Comment:
@josephperrott I believe all projection functions are going away in rxjs@^6, so this will likely have to change to a `map` operator.

Reply:
The deprecated API usages were removed in https://github.com/angular/material2/pull/10866.